pages/about/index.tsx

```diff
@@ -9,6 +9,32 @@ import { GetStaticProps, NextPage } from 'next';
 import { getServerSideTranslations } from 'utils/localization';
 import articles from '../../data/articles/articles.json';
 
+type Link = string;
+
+type Links = {
+  cs: Link;
+  sk: Link;
+  pl: Link;
+  uk: Link;
+};
+
+type key = string;
+
+const NAVIGATION: Record<key, Links> = {
+  stojimezaukrajinou: {
+    cs: 'https://www.stojimezaukrajinou.cz',
```

Comment:
Good, only please write basic URL always with ending "/"
Like: 
https://www.stojimezaukrajinou.cz/
https://cesko.digital/

Reply:
Ok, thank you.